src/wallet/rpcdump.cpp

```diff
@@ -1564,16 +1563,19 @@ static UniValue ProcessDescriptorImport(CWallet& wallet, const UniValue& data, c
             } else {
                 wallet.AddActiveScriptPubKeyMan(spk_manager->GetID(), *w_desc.descriptor->GetOutputType(), internal);
             }
+        } else {
+            if (w_desc.descriptor->GetOutputType()) {
+                wallet.DeactivateScriptPubKeyMan(spk_manager->GetID(), *w_desc.descriptor->GetOutputType(), internal);
+            }
```

Comment:
5d9670462219b8c753516bfbdc5c7fa23046cd07 does this have test coverage?

Reply:
This is covered in the section "Check can deactivate active descriptor". You can try deleting this lines and the test will fail